azure-pipelines.yml

```diff
@@ -19,6 +19,7 @@ resources:
     - repository: templates
       type: github
       name: bakdata/bakdata-project-templates
+      ref: tmp/upload-snapshot
```

Comment:
TODO: Remove after testing is finished

Reply:
@raminqaf Please revert and I will approve